tests/e2e/sample-applications/mongo-persistent/mongo-persistent-block.yaml

```diff
@@ -0,0 +1,267 @@
+apiVersion: v1
+kind: List
+items:
+  - kind: Namespace
+    apiVersion: v1
+    metadata:
+      name: mongo-persistent
+      labels:
+        app: todolist-mongo-go
+  - apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: mongo-persistent-sa
+      namespace: mongo-persistent
+      labels:
+        component: mongo-persistent
+  - kind: SecurityContextConstraints
+    apiVersion: security.openshift.io/v1
+    metadata:
+      name: mongo-persistent-scc
+    allowPrivilegeEscalation: true
+    allowPrivilegedContainer: true
+    runAsUser:
+      type: RunAsAny
+    seLinuxContext:
+      type: RunAsAny
+    fsGroup:
+      type: RunAsAny
+    supplementalGroups:
+      type: RunAsAny
+    volumes:
+    - '*'
+    users:
+    - system:admin
+    - system:serviceaccount:mongo-persistent:mongo-persistent-sa
+  - kind: ImageStream
+    apiVersion: image.openshift.io/v1
+    metadata:
+      name: todolist-mongo-go
+      namespace: mongo-persistent
+      labels:
+        app: todolist
+    spec: {}
+    status:
+      dockerImageRepository: ''
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      annotations:
+        template.alpha.openshift.io/wait-for-ready: 'true'
+      name: mongo
+      namespace: mongo-persistent
+      labels:
+        e2e-app: "true"
+    spec:
+      selector:
+        matchLabels:
+          app: mongo
+      strategy:
+        type: Recreate
+      template:
+        metadata:
+          labels:
+            e2e-app: "true"
+            app: mongo
+        spec:
+          serviceAccountName: mongo-persistent-sa
+          securityContext:
+            runAsUser: 0  
+          # Used to format the block device also touch one file which is used to determine when the
+          # block device was formatted. We use the same image as 
```

Comment:
Can you finish the end of this comment? Same image as ...?

Reply:
Done, reworded that as well.